dir.props

```diff
@@ -74,11 +74,12 @@
   <ItemGroup>
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/dotnet-core/" />
+    <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/roslyn-nightly/" />
```

Comment:
We should not add this feed dependency. Any packages we need from there should go on our buildtools feed.